qiskit/quantum_info/mitigation/base_readout_mitigator.py

```diff
@@ -0,0 +1,87 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Base class for readout error mitigation.
+"""
+
+from abc import ABC, abstractmethod
+from typing import Optional, List, Dict, Iterable, Tuple, Union, Callable
+import numpy as np
+from qiskit.result import Counts, QuasiDistribution
+
+try:
+    import matplotlib.pyplot as plt
+
+    _HAS_MATPLOTLIB = True
+except ImportError:
+    _HAS_MATPLOTLIB = False
+
+
+class BaseReadoutMitigator(ABC):
```

Comment:
We might want to include something in the base class for what physical qubits it is defined on.

Reply:
could you please elaborate?